src/util/pg.py

```diff
@@ -1770,6 +1770,7 @@ def get_m2m_tables(cr, table):
 
 class named_cursor(object):
```

Comment:
Maybe we should document this class and be explicit about it being a proxy object to psycopg2 server side cursors, adding only the convenient `dictfetch...` methods plus `iterdict`